en/user-guide/editing/editing.rst

```diff
@@ -36,42 +36,38 @@ Changing attributes of point elements (manholes / special structures)
 
  * To change an attribute you first need to select the **vw_qgep_wastewater_structure** layer.
  * Switch to the edit mode or start the **QGEP Wizard** and click on **Start Data Entry**.
- * Then click on the **Identify features** button and choose the element you want to edit by double-clicking.
+ * Then click on the **Identify features** button and click on the element you want to edit.
  
  .. figure:: images/identify_feature_tool.jpg
  
  * The **vw_qgep_wastewater_structure** form will open.
  
  .. figure:: ../digitizing/images/wizard_wastewater_structure_manhole_form_data_ok3.jpg
  
- * On the first tab you can change the most common attributes
+ * On the first tab (General) you can change the most common attributes
  * To change attributes of related tables (e.g. cover) you need to select the correspondent tab. Depending on the
    kind of wastewater structure you can edit additional attributes in the respective tab (**special structure**, **manhole**).
- * The **covers** tab allows you to add additional covers.
- 
- .. attention:: It is not recommended to add a second cover in this form, because you can not define the geometry of the new cover. See the :ref:`Adding-additional-covers-and-nodes-to-an-existing-wastewater-structure` chapter above.
  
  * In **Structure parts** tab you can add further parts such as **access aids**, **backflow prevention**, **dryweather flume** etc.
  
  .. figure:: images/form_vw_qgep_wastewater_structure_structure_parts.jpg
  
- * In **Wastewater nodes** tab you can edit or add all attributes of this class or add a second element.
- To add a second wastewater node in this part is not recommended, because you can not define the geometry of the new point.
-
- .. attention:: It is not recommended to add a second wastewater node in this form, because you can not define the geometry of the new node. See the :ref:`Adding-additional-covers-and-nodes-to-an-existing-wastewater-structure` chapter above.
+ * The **covers** and **wastewater nodes** tab allows you to look to all nodes or covers of a wastewater structure.
+ 
+ .. attention:: It is not recommended to add a second cover or wastewater node in the tabs of this form, because you can not define the geometry of the new points. See the :ref:`Adding-additional-covers-and-nodes-to-an-existing-wastewater-structure` chapter below.
```

Comment:
I think that is no longer true with QGIS 3.16, @urskaufmann should we remove this sentence?

Reply:
Yes, I will remove and mention this great Point-Child-Button